tiatoolbox/models/architecture/hovernet.py

```diff
@@ -544,45 +551,32 @@ def __proc_np_hv(np_map: np.ndarray, hv_map: np.ndarray):
         return proced_pred
 
     @staticmethod
-    # skipcq: PYL-W0221
-    def postproc(raw_maps: List[np.ndarray]):
-        """Post processing script for image tiles.
+    def _get_instance_info(pred_inst, pred_type=None):
+        """To collect instance information and store it within a dictionary.
 
         Args:
-            raw_maps (list(ndarray)): list of prediction output of each head and
-                assumed to be in the order of [np, hv, tp] (match with the output
-                of `infer_batch`).
+            pred_inst (np.ndarray): An image of shape (heigh, width) which
+                contains the probabilities of a pixel being a nuclei.
+            pred_type (np.ndarray): An image of shape (heigh, width, 1) which
+                contains the probabilities of a pixel being a certain type of nuclei.
 
         Returns:
-            inst_map (ndarray): pixel-wise nuclear instance segmentation
-                prediction.
-            inst_dict (dict): a dictionary containing a mapping of each instance
-                within `inst_map` instance information. It has following form
+            inst_info_dict (dict): A dictionary containing a mapping of each instance
+                    within `pred_inst` instance information. It has following form
 
-            >>> inst_info = {
-            >>>         box: number[],
-            >>>         centroids: number[],
-            >>>         contour: number[][],
-            >>>         type: number,
-            >>>         prob: number,
-            >>> }
-            >>> inst_dict = {[inst_uid: number] : inst_info}
+                >>> inst_info = {
```

Comment:
This should have a heading Examples:

Reply:
Bu this isn't an example usage, this is the form of the output. Would this still need the heading examples?

Reply:
Added example.